newIDE/app/src/Credits/CreditsStatusBanner.js

```diff
@@ -14,6 +14,7 @@ const styles = {
   container: {
     borderRadius: 8,
     padding: 8,
+    color: '#FAFAFA',
```

Comment:
Alternatively could have been in `gdevelopTheme.credits.textColor`?

Reply:
We already did it in different places (in app tutorial tooltip displayer for instance) and it feels more pragmatic like this